data/repos.yml

```diff
@@ -288,6 +288,12 @@
   type: Utilities
   production_hosted_on: eks
 
+- repo_name: govuk-design-guide
+  team: "#govuk-publishing-design-guide"
+  type: Publishing apps
```

Comment:
I don't think this should have it's own type, something like 'Design System'. Also the "Design System" team doesn't sit with in Publishing.

Reply:
This is design guidance for the publishing team so I think it's reasonable to stick it there. The team that contributes to that repository is in that Slack channel (I checked).